tests/template_tests/test_parser.py

```diff
@@ -45,6 +45,22 @@ def test_repr(self):
             '<Lexer template_string="{% for i in 1 %}{{ a...", verbatim=False>',
         )
 
+    def test_repr_multiline(self):
+        token = Token(TokenType.BLOCK, "some text")
+        self.assertEqual(repr(token), '<Block token: "some text...">')
+        parser = Parser([token], builtins=[filter_library])
+        self.assertEqual(
+            repr(parser),
+            '<Parser tokens=[<Block token: "some text...">]>',
+        )
+        filter_expression = FilterExpression("news|upper", parser)
+        self.assertEqual(repr(filter_expression), "<FilterExpression 'news|upper'>")
```

Comment:
Why are these part of this test? I don't think they have anything to do with newlines.

Reply:
You mean just those two lines? Yea I guess I just got them by basing the test on an existing test. Good point.

Reply:
I mean all nine lines I highlighted here.

Reply:
Oh. Yea, you're 100% correct. I'm confused. Sorry about that.